cleo/_utils.py

```diff
@@ -80,25 +81,31 @@ def find_similar_names(name, names):  # type: (str, List[str]) -> List[str]
     return suggested_names
 
 
-_TIME_FORMATS = [
-    (0, "< 1 sec"),
-    (2, "1 sec"),
-    (59, "secs", 1),
-    (60, "1 min"),
-    (3600, "mins", 60),
-    (5400, "1 hr"),
-    (86400, "hrs", 3600),
-    (129600, "1 day"),
-    (604800, "days", 86400),
+@dataclass
+class TimeFormat:
+    threshold: int
+    alias: str
+    divisor: Optional[int] = None
+
+    def apply(self, secs: float) -> str:
+        if self.divisor:
+            return f"{math.ceil(secs / self.divisor)} {self.alias}"
+        return self.alias
+
+
+_TIME_FORMATS: List[TimeFormat] = [
+    TimeFormat(0, "< 1 sec"),
+    TimeFormat(2, "1 sec"),
+    TimeFormat(59, "secs", 1),
+    TimeFormat(60, "1 min"),
+    TimeFormat(3600, "mins", 60),
+    TimeFormat(5400, "1 hr"),
+    TimeFormat(86400, "hrs", 3600),
+    TimeFormat(129600, "1 day"),
+    TimeFormat(604800, "days", 86400),
 ]
 
 
-def format_time(secs):  # type: (float) -> str
-    for fmt in _TIME_FORMATS:
-        if secs > fmt[0]:
-            continue
-
-        if len(fmt) == 2:
-            return fmt[1]
-
-        return f"{math.ceil(secs / fmt[2])} {fmt[1]}"
+def format_time(secs: float) -> str:
+    format = next(fmt for fmt in _TIME_FORMATS if secs < fmt.threshold)
```

Comment:
Ah, we've found a bug! The original would return None for > 604800 seconds

```suggestion
    format = next((fmt for fmt in _TIME_FORMATS if secs < fmt.threshold),  _TIME_FORMATS2[-1])
```